gpu-aware-scheduling/pkg/gpuscheduler/scheduler.go

```diff
@@ -453,8 +465,7 @@ func (m *GASExtender) checkGpuAvailability(gpuName string, node *v1.Node, pod *v
 	}
 
 	if !gpuMap[gpuName] {
-		klog.Warningf("node %v gpu %v has vanished", node.Name, gpuName)
-
+		// this can happen e.g. during same-gpu usage when the i915-resource is running out
```

Comment:
wouldn't be good to keep the warning or with a different verbosity level with the new msg?

Reply:
No, the fact that the map can end up pruned is being leveraged in different features going forward, starting now. So the warning needs to be removed. It's a feature now.